lib/global_uid.rb

```diff
@@ -16,19 +16,20 @@ class TimeoutException < Exception ; end
 ActiveRecord::Base.send(:include, GlobalUid::ActiveRecordExtension)
 ActiveRecord::Migration.send(:prepend, GlobalUid::MigrationExtension)
 
+# Make sure that GlobalUID is disabled for ActiveRecord's SchemaMigration table
 if defined?(ActiveRecord::SchemaMigration)
   ActiveRecord::SchemaMigration.disable_global_uid
 end
 
+# Make sure that GlobalUID is disabled for ActiveRecord's Internal Metadata table
+if ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
We should probably change the `defined?(ActiveRecord::SchemaMigration)` to use a Rails version. Currently we could have a bug if global_uid is loaded before this class is defined.